posts/Davide/2018-05-25-hibernate-ogm-5-4-Beta1-released.adoc

```diff
@@ -0,0 +1,83 @@
+= "Hibernate OGM 5.4.0.Beta1 release"
+Davide D'Alto
+:awestruct-tags: [ "Hibernate OGM", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+http://hibernate.org/ogm/releases/5.4/#get-it[Hibernate OGM 5.4.0.Beta1] has been released!
+
+The main differences from the previous release is the upgrade to Hibernate ORM 5.3.0.Final, 
+the support for Infinispan Remote native queries and the use of cluster counters
+for local caches when generating sequences with Infinispan Embedded.
```

Comment:
Maybe for local cache **too** (for remote it was already provided)

Reply:
I don't think it's necessary to add the `too`, the fact the we added support for one case doesn't mean it's not already supported for other use cases.